json/codec.go

```diff
@@ -1046,8 +1065,12 @@ var (
 	rawMessagePtrType = reflect.PtrTo(rawMessageType)
 
 	sliceInterfaceType      = reflect.TypeOf(([]interface{})(nil))
+	sliceStringType         = reflect.TypeOf(([]interface{})(nil))
 	mapStringInterfaceType  = reflect.TypeOf((map[string]interface{})(nil))
 	mapStringRawMessageType = reflect.TypeOf((map[string]RawMessage)(nil))
+	mapStringStringType     = reflect.TypeOf((map[string]string)(nil))
+	mapStringStringSliceType    = reflect.TypeOf((map[string][]string)(nil))
+	mapStringBoolType       = reflect.TypeOf((map[string]bool)(nil))
```

Comment:
`go fmt` to align these?

Reply:
Oh, gopls doesn't auto-format after renaming variables :(